pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,536 @@
+---
+title: GraphQL Validations migration guide
+---
+
+Soon `graph-node` will support 100% coverage of the [GraphQL Validations specification](https://spec.graphql.org/June2018/#sec-Validation).
```

Comment:
We should align on timing and be clear in this document what the expectations are for:
- Hosted Service
- A release on the network